geopandas/sindex.py

```diff
@@ -168,41 +168,62 @@ def query_bulk(self, geometry, predicate=None, sort=False):
         """
         raise NotImplementedError
 
-    def intersection(self, coordinates):
-        """Compatibility wrapper for rtree.index.Index.intersection,
-        use ``query`` instead.
+    def nearest(self, geometry):
```

Comment:
I have switched the order of methods in the code to have `nearest` and `nearest_all` together, that is why this messy diff.